src/v2/view-builder/views/ov/EnrollPollOktaVerifyView.js

```diff
@@ -71,6 +67,16 @@ const Body = BaseForm.extend(Object.assign(
 
       }
     },
+    postRender() {
```

Comment:
there is an `afterRender` event that is triggered in FormController, would that help instead of adding a `setTimout` here?

Reply:
but for is not aware of existence of the controller and hence cannot listen to the event.